### PR TITLE
B/ss-e2e

### PR DESCRIPTION
**Pull request by rej156**
- [x] Fix standard e2e tests
- [x] Port stableswap e2e tests

### CODE REVIEW

**Comment by rej156** [comment]
To run locally too,
```
yarn cypress install
yarn build
yarn integration-test
```

**Comment by rej156** [comment]
Screenshot 2022-03-27 at 23 26 07

File: cypress/integration/remove-liquidity.test.ts [hunk 1]
```diff
@@ -1,39 +1,99 @@
 describe('Remove Liquidity', () => {
   it('redirects', () => {
-    cy.visit('/remove/0xc778417E063141139Fce010982780140Aa0cD5Ab-0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85')
+    cy.visit('/remove/0xC9BdeEd33CD01541e1eeD10f90519d2C06Fe3feB-0x1d1f82d8b8fc72f29a8c268285347563cb6cd8b3')
```
Comment (specialist0x) [inline cypress/integration/remove-liquidity.test.ts:3]: Should these addresses be imported `const`s ?

File: cypress/integration/remove-liquidity.test.ts [hunk 2]
```diff
@@ -1,39 +1,99 @@
...
-    cy.get('#remove-liquidity-tokenb-symbol').should('contain.text', 'MKR')
+    cy.visit('/remove/0x7821c773a12485b12a2b5b7bc451c3eb200986b1-0x1d1f82d8b8fc72f29a8c268285347563cb6cd8b3')
+    cy.get('#remove-liquidity-tokena').should('contain.text', 'SUSHI')
+    cy.get('#remove-liquidity-tokenb').should('contain.text', 'MKR')
+  })
+
+  it('loads the USDT_USDC stableswap pool by name', () => {
+    cy.visit('/pool/stable/remove/USDT_USDC')
+    cy.url().should('contain', '/pool/stable/remove/USDT_USDC')
+    cy.get('#remove-liquidity-input-tokena .token-symbol-container').should('contain.text', 'USDT')
```
Comment (specialist0x) [inline cypress/integration/remove-liquidity.test.ts:43]: Should this come from the imported token const's `symbol` field?

File: src/pages/StableSwapPoolRemoveLiquidity/StableSwapRemoveLiquidityCurrencyRow.tsx [hunk 3]
```diff
@@ -11,13 +11,14 @@ import _ from 'lodash'
...
+  index: number
 }
 
-export default function StableSwapRemoveCurrencyRow({ currency, value }: Props) {
+export default function StableSwapRemoveCurrencyRow({ currency, value, index }: Props) {
   const { t } = useTranslation()
 
   return (
-    <InputPanel>
+    <InputPanel id={`remove-liquidity-input-token${['a', 'b', 'c'][index]}`}>
```
Comment (specialist0x) [inline src/pages/StableSwapPoolRemoveLiquidity/StableSwapRemoveLiquidityCurrencyRow.tsx:21]: nit I'd changed the field names from `a/b/c` to reflect their index positions on the solidity contract -- if it's easier you can do `remove-liquidity-input-token[index]`

ndb tho this is working fine, too

**Review by specialist0x — APPROVED** [review]
This diff is so fire!!

File: src/pages/StableSwapPoolRemoveLiquidity/StableSwapRemoveLiquidityCurrencyRow.tsx (hunk 3 above)
Comment (rej156) [reply src/pages/StableSwapPoolRemoveLiquidity/StableSwapRemoveLiquidityCurrencyRow.tsx:21]: will keep it if that's cool just cause it's the same as the default amm remove liquidity too

they're just ids anyways so it's just this

**Pull request merged**